Resources/Prototypes/Entities/Clothing/Neck/cloaks.yml

```diff
@@ -67,7 +67,7 @@
 - type: entity
   parent: ClothingNeckBase
   id: ClothingNeckCloakQm
-  name: logistics officer's cloak # DeltaV - Logistics Department replacing Cargo
+  name: cargo technician's cloak
```

Comment:
This can probably go unchanged- or perhaps be labeled QM? Cargo Techs don't get cloaks :P